src/Features/Core/Portable/Structure/BlockStructureOptions.cs

```diff
@@ -11,6 +11,9 @@ namespace Microsoft.CodeAnalysis.Structure
         bool ShowOutliningForCommentsAndPreprocessorRegions = true,
         bool ShowOutliningForDeclarationLevelConstructs = true,
         bool ShowOutliningForCodeLevelConstructs = true,
+        bool CollapseRegionsOnFileOpen = true,
+        bool CollapseUsingsOnFileOpen = false,
```

Comment:
CollapseImportsOnFileOpen

Reply:
Perhaps CollapseRegionsWhenFirstOpened?  Because we won't always do this right?  Just hte first time, at which point the user state for open/closed will be preserved and respected for future openings?

Reply:
Yeah, just the first time. I was mirroring the existing "Enter outlining mode when files open", though granted I think thats terrible wording too :)